drivers/include/periph/eeprom.h

```diff
@@ -73,6 +73,35 @@ void eeprom_write_byte(uint32_t pos, uint8_t data);
  */
 size_t eeprom_write(uint32_t pos, const uint8_t *data, size_t len);
 
+/**
+ * @brief   Set @p len bytes from the given position @p pos with value @p val
+ *
+ * @param[in] pos       start position in eeprom
+ * @param[in] len       the number of bytes to set
+ *
```

Comment:
Should you add param[in] val to the doxygen param list as well?

Reply:
Is `[in]` even necessary. Is it not the default?

Reply:
fixed

> Is [in] even necessary? Is it not the default?

Following [Python's philosophy](https://www.python.org/dev/peps/pep-0020/#id3), I prefer when it's explicit (rather than implicit)